### PR TITLE
Update notepad.md

### PR DESCRIPTION
**Pull request by pndda**

### CODE REVIEW

**Comment by RafSwiggers** [comment]
Noice.

**Review by pndda — COMMENTED** [review]
Deleted 'NOT NULL' from ID because it's AUTO_INCREMENTED. Will never be null 
Title is unique because you have to search on title

**Review by SeppeVerhavert — COMMENTED** [review]
Why start with create table and not create database?
Why can your title be 255 characters long?
Why do you use TEXT for textbody?

File: notepad.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+CREATE TABLE notes (
+ID INT PRIMARY KEY AUTO_INCREMENT , 
+title varchar(255) UNIQUE NOT NULL, 
+textbody text
```
Comment (nicplackle) [inline notepad.sql]: Use 'varchar(MAX)' instead of 'text'. The text datatype is deprecated and should not be used for new development work.

**Comment by pndda** [comment]
Changed Varchar from 'title' to 50. 
Textbody is TEXT because, you will get to write more characters, when varchar is limited to 255

File: notepad.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+CREATE DATABASE API_notes;
+
+CREATE TABLE notes (
+ID INT NO NULL PRIMARY KEY AUTO_INCREMENT , 
+title varchar(50) UNIQUE NOT NULL, 
+textbody text(1000)
+)
```
Comment (Ilger) [inline notepad.sql]: Statments always end with a semicolon `;`

File: notepad.sql [hunk 3]
```diff
@@ -1,6 +1,7 @@
+CREATE DATABASE API_notes;
+
 CREATE TABLE notes (
 ID INT NO NULL PRIMARY KEY AUTO_INCREMENT ,
```
Comment (YelenaMerzlyakova) [inline notepad.sql]: why auto increment?

File: notepad.sql [hunk 4]
```diff
@@ -1,6 +1,7 @@
+CREATE DATABASE API_notes;
+
 CREATE TABLE notes (
```
Comment (YelenaMerzlyakova) [inline notepad.sql]: Why do you use a plural name for your table?

**Review by Ilger — CHANGES_REQUESTED** [review]
Statments always end with a semicolon ;

File: notepad.sql (hunk 2 above)
Comment (pndda) [reply notepad.sql]: ok

File: notepad.sql (hunk 4 above)
Comment (pndda) [reply notepad.sql]: because, i want to get multiple notes in my table

**Pull request merged**